package.json

```diff
@@ -9,6 +9,7 @@
   "private": true,
   "dependencies": {
     "@prisma/client": "4.11.0",
+    "fuzzysort": "2.0.4",
```

Comment:
This library gives us a sorted list of matches for a query string. In the future, we could use this list of matches to populate an auto-complete style suggestion dropdown.